### ISSUE TITLE
M-net pre-train

### ISSUE THREAD

**Issue by ulnay** [lizhengwei1992/Semantic_Human_Matting#15]
First of all, thank you very much for implementation and sharing. 
I just got one question,  in the paper, section 4.5 , they've mentioned they also have M-net pretrain, in your implementation, how can we pre-train the M-net?
Thank you!

**Comment by Jason-xin** [comment]
the same question

**Comment by rainsun1** [comment]
1. T-Net pre-train: the model shoule only include the t-net part, and you can only train the model with L_t loss. 
2. M-Net pretrain: the model shoule only include the m-net part, and the input must include 6 mixed channels including 3-channel image and 3-channel trimap(fg, bg, unsure), and feed them to m-net. L_alpha and L_composition losses are included here.
3. Fine tune: include both parts, with the combined two weights above.
